app/views/settings/edit.html.erb

```diff
@@ -11,17 +11,35 @@
       <%= select_tag :level, options_for_select(@mfa_options, @user.mfa_level), class: "form__input form__select" %>
 
       <div class="text_field">
-        <%= label_tag :otp, "OTP code", class: "form__label" %>
+        <%= label_tag :otp, t(".otp_code"), class: "form__label" %>
         <%= text_field_tag :otp, "", class: "form__input", autocomplete: :off %>
       </div>
       <%= submit_tag t(".mfa.update"), class: "form__submit" %>
     <% end %>
   <% else %>
-    <p>
+    <div class="t-body">
+      <h2><%= t ".mfa.multifactor_auth" %></h2>
```

Comment:
<img width="1050" alt="Screen Shot 2022-09-19 at 12 24 42 PM" src="https://user-images.githubusercontent.com/42748004/191065908-84b3892e-78ef-4568-b8f5-1ef56510a3af.png">

Not a big fan of the duped multifactor auth heading. Could we have a top level heading of Multi-factor auth and sub headings of TOTP/App MFA and Security device MFA?

In this or a followup PR, I could see this section be promoted to its individual page (like API keys). wdyt?

Reply:
I've updated the headings for this page. I can follow up in the future with splitting it out into its own page 
<img width="411" alt="image" src="https://user-images.githubusercontent.com/8496209/192001211-b1e66b56-311b-42ad-b085-c46caecb85cf.png">

Reply:
I'd like to put splitting it out onto a separate page as a new PR as there are a few dependencies to change (like MFA redirects for new/weak level) that i think will add a bit of bulk to this PR if we do it now